models/MLmodels/DataReader.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+from IPython.display import display
+from sklearn.model_selection import train_test_split
+from visualizers import model_plotter
+
+class DataSample:
```

Comment:
Alla modeller använder datan på en viss struktur som implicerar att den måste vara i en klass likt denna. För att man ska kunna trycka in data på ett dynamiskt sätt skapa gärna en klass utöver den här där man som argument i konstruktorn kan skicka in X och Y, och att det enda som klassen sen gör är att shuffla, splitta och spara på objektet för att användningen ska fungera i modeller.